go/vt/sqlparser/sql.y

```diff
@@ -699,7 +699,7 @@ common_table_expression:
 union_lhs:
   base_select
   {
-    if _, ok := $1.HasIntoDefined(); ok {
+    if $1.GetInto() == nil {
```

Comment:
Should these next three instances be `!=` instead of `==`? Good chance to check if we have tests that fired, and if not, see if there are test gaps we should fix.

Reply:
yea, I made typos and it was caught in the previous commit. I fixed it in the latest commit.